api/response/models.py

```diff
@@ -34,3 +34,9 @@ class Response(Base, Utility):
         secondary="missing_items",
         backref=('resources'),
         lazy="joined")
+
```

Comment:
Hey @RonKbS , I think you should calculate the responses in the room query in the schema instead of having this property in the models. If done in the schema it will reduce the boilerplate.

Reply:
Also, I strongly feel that this could have been approached best by refactoring `question/schema_query.py`. There's part of this functionality there, the problem is that the query did not take a question id yet. Else we could just do away with the whole file because it is not used at all. Thoughts?

Reply:
Doing this in the schema will not make the data available to other queries because I am taking advantage of relations to access the responses from the questions. Also, I do not think this is (boilerplate)[https://en.wikipedia.org/wiki/Boilerplate_code] because the code is not similar to other pieces in the codebase.

Reply:
`Else we could just do away with the whole file because it is not used at all. Thoughts?` ?

Reply:
@bevkololi the refactoring of that file would require much more time and this PR has been raised to unblock an urgent task in the frontend. Welike Amos can be contacted for more information on this.